pkg/controllers/management/nodetemplate/nodetemplate.go

```diff
@@ -22,7 +22,7 @@ import (
 )
 
 const (
-	ownerBindingsAnno = "ownerBindingsCreated"
+	OwnerBindingsAnno = "ownerBindingsCreated"
```

Comment:
Why does GH mark this file as changed?  I do not see anything is changed.

Reply:
The `o` in `ownerBindingsAnno` is now capitalized so it can be used in other packages.